CONTRIBUTING.md

```diff
@@ -1,151 +1,192 @@
 # Contributing to osquery
 
-We want to make contributing to osquery as simple and transparent as possible. These guidelines
-explain the basics of the osquery development process and how you can contribute. Please read
-these guidelines before submitting your code as they are designed to save you time later on when
-your code is under review.
+We want to make contributing to osquery as simple and transparent as
+possible. These guidelines explain the basics of the osquery
+development process and how you can contribute. Please read these
+guidelines before submitting your code as they are designed to save
+you time later on when your code is under review.
 
 ## Contributing 101
 
-All contributions are submitted via pull requests (PRs) open against the osquery's
-[master](https://github.com/osquery/osquery/tree/master) branch on
-GitHub. After being reviewed by the _core team_ and tested by CI, if all
-is well, they will be pushed to master and the corresponding PR closed.
-
-You can see who the _core team_ is by viewing the [team page](https://github.com/orgs/osquery/teams)
-on the osquery GitHub organization.
-
-If you need help, both the core team and community members are on the osquery
-[Slack](https://osquery.slack.com). Feel free to register using the following
-[link](https://slack.osquery.io/) if you haven't done so yet and get in touch with us.
-The `#code-review` Slack channel has been set up to handle urgent review needs as well as questions
-about your PR. Note: prefer to keep discussion about code changes in the GitHub
-pull request thread.
-
-The osquery team also hosts regular office hours where the community is invited to discuss osquery
-development with the core team. You are welcome to join. Office hours are announced on our Slack on
-the `#officehours` channel.
+All contributions are submitted via pull requests (PRs) open against
+the osquery's [master](https://github.com/osquery/osquery/tree/master)
+branch on GitHub. After being reviewed by the _core team_ and tested
+by CI, if all is well, they will be pushed to master and the
+corresponding PR closed.
+
+You can see who the _core team_ is by viewing the [team
+page](https://github.com/orgs/osquery/teams) on the osquery GitHub
```

Comment:
I think this is not viewable to people outside of the teams :)

Reply:
Hrm. That link should be https://github.com/orgs/osquery/teams/technical-steering-committee and I'm not sure. It's flagged as a visible team

Reply:
That is a 404 as well

Reply:
Oh, I see now. This is a different part of that doc. There is no core team right now.